src/components/log/ProfileLogView.js

```diff
@@ -19,7 +29,9 @@ class ProfileLogView extends PureComponent {
             .filter(markerIndex => {
               const data = markers.data[markerIndex];
               return (
-                data !== null && 'category' in data && data.category === 'log'
+                data !== null &&
+                data.type === 'tracing' &&
```

Comment:
@mstange can you double check my suggestion here? Flow really doesn't infer a whole lot from the `x in y` expressions. I don't know if we have documented how to activate the logging feature. Do you have any saved profiles with logs?